mslib/utils/mssautoplot.py

```diff
@@ -316,67 +429,124 @@ def draw(self):
 @click.option('--intv', default=0, help='Time interval.')
 @click.option('--stime', default="", help='Starting time for downloading multiple plots with a fixed interval.')
 @click.option('--etime', default="", help='Ending time for downloading multiple plots with a fixed interval.')
-def main(cpath, view, ftrack, itime, vtime, intv, stime, etime):
+@click.pass_context
+def main(ctx, cpath, view, ftrack, itime, vtime, intv, stime, etime):
+    if ctx.obj is not None:
+        pdlg = QProgressDialog("Downloading images", "Cancel", 0, 10, parent=ctx.obj)
```

Comment:
ToDo: This does not work headless. We need a different solution

Reply:
Correction: it works headless, but  it is rather expensive by requiring QT for a progressbar installed